frame/identity/src/lib.rs

```diff
@@ -238,6 +238,8 @@ pub mod pallet {
 		NotOwned,
 		/// The provided judgement was for a different identity.
 		JudgementForDifferentIdentity,
+		/// Error thrown when repatriate_reserved returns an error.
+		RepatriateReservedError,
```

Comment:
```suggestion
		JudgementPaymenrFailed,
```

Also change doc to make this more clear

Reply:
> Also change doc to make this more clear

I noticed there is a typo in your suggested change @bkchr